docs/resources/serving_endpoint.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "MLflow"
+---
+# databricks_serving_endpoint Resource
+
+This resource allows you to manage [Model Serving Endpoints](https://docs.databricks.com/machine-learning/model-inference/serverless/serverless-real-time-inference.html) in Databricks.
```

Comment:
Should we link to the REST APIs that are being wrapped by terraform somewhere? (Feels like we are duplicating a lot of the field-specific documentation, not sure if there's a simple way to solve this.)